scripts/getDependencies.pl

```diff
@@ -94,6 +109,16 @@
 		fname => 'junit4.jar',
 		sha1 => 'e4f1766ce7404a08f45d859fb9c226fc9e41a861'
 	},
+	junit_jupiter_api => {
```

Comment:
This is not needed for PR https://github.com/adoptium/aqa-tests/pull/5133. Please remove.